topgg.rc

```diff
@@ -0,0 +1,33 @@
+#include <windows.h>
```

Comment:
is it still possible to compile this library static on windows?

if so you definitely don't want to bring in an .RC file as it's definitions will conflict with those in the project and set this version info for the projects main executable.

Reply:
well, i found that dl.dpp.dev only has .dll files... so i thought it was not possible to create a static wrapper over D++ haha

Reply:
i think this should be resolved by [this commit](https://github.com/Top-gg-Community/cpp-sdk/pull/10). what do you think? feel free to give feedback! @braindigitalis